docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:14.04
+MAINTAINER svanoort <samvanoort@gmail.com>
```

Comment:
NIT/holywar: Should we add license headers and corporate e-mails in the case of such changes? I don't care much, but there was a related discussion in other Jenkins component several weeks ago